### PR TITLE
Fix videos saved as root on Linux when `docker run -v`

### PR DESCRIPTION
**Pull request by elgalu**
We can see the current problem in Travis for example:

image

Fixing ... WIP ...

### CODE REVIEW

File: scripts/entry.sh [hunk 1]
```diff
@@ -28,11 +28,36 @@ else
...
-# Make sure Docker works before continuing
-docker --version
-sudo docker images elgalu/selenium >/dev/null
-
-# To run docker alongside docker we still need sudo inside the container
-# TODO: this can potentially be fixed perhaps by passing $UID
-#       during docker run time.
-exec sudo --preserve-env ./zalenium.sh "$@"
+# If this was docker run with: -e HOST_GID="$(id -g)" -e HOST_UID="$(id -u)"
+if [ ${HOST_UID} != "" ] && [ ${HOST_GID} != "" ]; then
```
Comment (diemol) [inline scripts/entry.sh]: I get `/usr/bin/entry.sh: line 32: [: !=: unary operator expected`

Comment (diemol) [inline scripts/entry.sh]: 😸

**Comment by elgalu** [comment]
:+1:

**Comment by diemol** [comment]
👍

**Pull request merged**